Can't require() react-tools module

I'm trying to programatically invoke the JSX transformer (using the version of `react-tools` in the npm registry) by running something like

```
require('react-tools').transform(someCode);
```

Which then throws this error:

```
Error: Cannot find module './build/React'
```

If I comment out the lines in `main.js` that require/use `./build/React` the `react-tools` module loads fine and the transformer runs correctly.

--- a/main.js
+++ b/main.js
@@ -1,6 +1,6 @@
 'use strict';
 
-var React = require('./build/React');
+var React = require('./build/react');
 var visitors = require('./vendor/fbtransform/visitors').transformVisitors;
 var transform = require('./vendor/fbtransform/lib/transform').transform;
 

--- a/package.json
+++ b/package.json
@@ -16,7 +16,7 @@
     "README.md",
     "main.js",
     "bin/jsx",
-    "build/React.js",
+    "build/react.js",
     "vendor/fbtransform/",
     "vendor/woodchipper.js"
   ],

